src/colmap/sensor/models_test.cc

```diff
@@ -228,5 +228,26 @@ TEST(ThinPrismFisheye, Nominal) {
                                           0.001});
 }
 
+TEST(AriaFisheye, Nominal) {
+  std::vector<double> params = {241.996,
+                                241.996,
```

Comment:
I suggest using a different value for fx/fy so we can detect possible bugs when accidentally swapping coordinates. Thanks.